src/views/Main.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import CountryCard from '../Components/CountryCard';
+import './Main.css';
+import fetchCountries from '../services/Countries';
+import Dropdown from '../Components/Dropdown';
+
+export default function Main() {
+  const [countries, setCountries] = useState([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [continent, setContinent] = useState('All');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const resp = await fetchCountries();
+        setCountries(resp);
+        setTimeout(() => {
+          setLoading(false);
+        }, 850);
+      } catch (e) {
+        setErrorMessage('Ooooopsie Daisys girl.2.0! Refresh page please!');
+      }
```

Comment:
Great use of a try/catch block here to render the error message! Nice job Yovana!